### DISCUSSION TITLE
项目启动mfsu配置后，编译报错问题

### DISCUSSION THREAD

**Discussion by yangda666** [umijs/umi#7905] [Q&A]
ERROR in ./node_modules/react-dnd/dist/esm/index.js 2:0-24
Module not found: Error: Can't resolve './hooks' in 'D:\work\khaos-web\node_modules\react-dnd\dist\esm'
Did you mean 'index.js'?
BREAKING CHANGE: The request './hooks' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./src/.umi/.cache/.mfsu/mf-va_react-dnd.js 1:0-26 1:0-26
 @ container entry ./react-dnd[0]

**Comment by sogud** [comment]
同样遇到

**Comment by sogud** [comment]
关了mfsu解决，或者配置nodeModulesTransform: {
    type: 'none',
    exclude: [],
  },
